utils/post_utils.jsx

```diff
@@ -122,6 +124,17 @@ export function containsAtChannel(text, options = {}) {
     return (/\B@(all|channel)\b/i).test(mentionableText);
 }
 
+export const groupsMentionedInText = (text, groups) => {
+    // Don't warn for slash commands
+    if (!text || text.startsWith('/')) {
+        return [];
+    }
+
+    const mentionableText = formatWithRenderer(text, new MentionableRenderer());
+    const mentions = allAtMentions(mentionableText);
+    return mentions.length > 0 && groups.filter((group) => mentions.includes(`@${group.name}`));
```

Comment:
specifically talking about optimizing this function here right now it would be quite inefficient with a large list of mentions and a large list of groups. 

could replace it with something like `mentions.filter((mention) => groups[mention]));`

Reply:
Implemented! Thanks